yt_dlp/YoutubeDL.py

```diff
@@ -2244,9 +2259,7 @@ def _calc_headers(self, info_dict):
         return res
 
     def _calc_cookies(self, url):
-        pr = sanitized_Request(url)
-        self.cookiejar.add_cookie_header(pr)
-        return pr.get_header('Cookie')
+        return get_cookie_header(Request(url), self.cookiejar)
```

Comment:
as mentioned above, it makes a lot of sense for this to be just:
```suggestion
        return self.cookiejar.get_cookies_of(url)
```